src/combineReducers.js

```diff
@@ -95,6 +95,11 @@ export default function combineReducers(reducers) {
   var finalReducers = {}
   for (var i = 0; i < reducerKeys.length; i++) {
     var key = reducerKeys[i]
+    if (process.env.NODE_ENV !== 'production') {
+      if (typeof reducers[key] === 'undefined') {
+        warning(`No reducer provided for key "${key}"`)
+      }
+    }
```

Comment:
Nit: Do you mind adding newlines before and after this new block? It makes it easier to spot the dev-only code.